manifests/startup.pp

```diff
@@ -41,8 +43,10 @@
       fail('we currently only support a title that contains agent or server')
     }
   }
-  # provided by camp2camp/systemd
   if $facts['systemd'] {
+    if $name =~ /^zabbix-agent2?$/ {
```

Comment:
Hello @ekohl ,

I'm sorry, I don't really use Puppet anymore since 3-4 month. Maybe someone can be fork my branch or looking that to another PR ?

Regards,